vllm/model_executor/models/qwen2_5_vl.py

```diff
@@ -296,20 +302,23 @@ def forward(
                                       "(b s) ... -> b s ...",
                                       b=batch_size)
         elif self.attn_backend == _Backend.TORCH_SDPA:
-            seq_length = q.size(1)
-            q, k, v = (rearrange(x, "b s h d -> b h s d") for x in [q, k, v])
-            attention_mask = torch.zeros([1, seq_length, seq_length],
-                                         device=q.device,
-                                         dtype=torch.bool)
+            # Remove attention_mask for speed & less VRAM.
```

Comment:
Let's change this comment to something like below for less confusion.
```suggestion
            # Execute attention entry by entry for speed & less VRAM.
```

Reply:
> Can you resolve these comments as well?

@DarkLight1337  done!